swagger.yaml

```diff
@@ -844,4 +1006,53 @@ components:
         lastUpdatedAt:
           type: string
           format: date-time
-          description: Date and time when the software entry was last updated
\ No newline at end of file
+          description: Date and time when the software entry was last updated
+        
+    Questionnaire:
+      type: object
+      properties:
+        id:
+          type: integer
+          description: Unique identifier for the questionnaire
+        title:
+          type: string
+          description: Title of the questionnaire
+        description:
+          type: string
+          description: Description of the questionnaire
+        options:
+          type: array
+          items:
+            $ref: '#/components/schemas/Questions'
+          description: List of questions in the questionnaire
+        tags:
+          type: array
+          items:
+            type: string
+          description: List of tags for the questionnaire
+        passedUsers:
+          type: array
+          items:
+            type: integer
+          description: List of users who have passed the questionnaire
+        passedScore:
+          type: integer
+          description: Minimum score required to pass the questionnaire
+
+    Questions:
```

Comment:
It would be nice if the questionnaire to begin with can handle both single select and mutli select. Does the current schema allow this?